source/docs/prismic/integration-fields.md

```diff
@@ -0,0 +1,289 @@
+---
+id: integration-fields
+title: Integration fields
+---
+
+Prismic [Integration fields](https://prismic.io/feature/integration-field) are custom fields for retrieving content from external systems. When using Prismic with Front-Commerce, you might want to create Integration fields for Categories, Products or other custom entities in order to improve the Content Management experience.
+
+**Enable Content managers to build a landing page by selecting Products retrieved from your eCommerce backend without leaving the writing room** for instance. As developer, seamlessly access the related Product data in GraphQL so that you can leverage and reuse existing GraphQL fragments and UI components.
```

Comment:
```suggestion
**Enable Content managers to build a landing page by selecting Products retrieved from your eCommerce backend without leaving the writing room**. As a developer you can seamlessly access the related Product data in GraphQL so that you can leverage and reuse existing GraphQL fragments and UI components.
```